.env.example

```diff
@@ -0,0 +1,9 @@
+
+# Note we depend on NODE_ENV being set to dictate which of the env variables below get loaded at runtime. 
+
+# Application Port - express server listens on this port (default 3000).
+PLAYGROUND_PORT=3000
+
+# URL of the AskScript backend, including path
+PLAYGROUND_ASK_SERVER_URL="http://cli.askql.org:6001/askscript"
+PLAYGROUND_ASK_SERVER_VERSION_URL="http://cli.askql.org:6001/version"
```

Comment:
Maybe it would be easier to just have url without path? We might want more flexibility with endpoints

Reply:
How is the port configured for askql server? Why is it `6001` here if "PLAYGROUND_PORT" is `3000`?